src/test/scala/edu/berkeley/ce/sparkrocks/SeedJointSelectorSpec.scala

```diff
@@ -179,8 +158,80 @@ class SeedJointSelectorSpec extends FunSuite {
     )
     val generatedInput = JointGenerator(globalOrigin, boundingBox, rockVolume, jointData)
     val initialBlock = Block(generatedInput.origin, generatedInput.rockVolume)
-    val (seedJoints, _) = SeedJointSelector.searchJointSets(generatedInput.jointSets, initialBlock, 4)
+    val (seedBlocks, _) = SeedJointSelector.generateSeedBlocks(generatedInput.jointSets, initialBlock, 3)
+
+    assert(seedBlocks.length >= 3)
+  }
+
+  test("Select 2 seed joints from multiple joint sets") {
```

Comment:
You know it! I like to write code that doesn't work first time round, so this is an absolute necessity.